src/main/charts/bitbucket/templates/rbac.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and (.Values.bitbucket.clustering.enabled) (or .Values.serviceAccount.clusterRole.create .Values.serviceAccount.role.create) -}}
+apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.serviceAccount.clusterRole.create }}
+kind: ClusterRole
+{{- else }}
+kind: Role
+{{- end }}
+metadata:
+  name: {{ template "bitbucket.clusterRoleName" . }}
```

Comment:
I know we want to keep the changes limited but what about renaming this to `bitbucket.roleName`?

Reply:
Honestly, I don't even understand why we have `name` in there and what's the benefit of letting the user to override it. And yes, my motivation was to reuse as much as possible.

Reply:
@nanux ok, I'm now using `{{ include "common.names.fullname" . }}` for role and rolebinding names to avoid confusion. Like I said before, I do not see much value in introducing name override here.